nucliadb_node/src/settings/providers.rs

```diff
@@ -150,26 +150,54 @@ pub mod env {
         #[test]
         #[serial]
         fn test_default_env_settings() {
+            // Safe current state of DATA_PATH
```

Comment:
nit: you could add a function that does this copy-and-restore work to avoid repetition

Reply:
I just did this to have something that would solve the env var mess, I would not want to loose much time in this approach because I dont think is the way to go.
In a separate PR I will try to centralise the way tests get env vars so doing this set-and-restore is not needed.